src/tribol/integ/Integration.cpp

```diff
@@ -79,19 +79,23 @@ TRIBOL_HOST_DEVICE void EvalWeakFormIntegral<COMMON_PLANE, SINGLE_POINT>( Surfac
                            elem.overlapNormal[2], cx[0], cx[1], cx[2], projX1[elem.dim * i], projX1[elem.dim * i + 1],
                            projX1[elem.dim * i + 2] );
 
+#ifdef TRIBOL_USE_HOST
```

Comment:
Just confirm whether this should also check to see if tribol_debug is defined. You added TRIBOL_DEBUG, but not all SLIC_DEBUG statements are guarded.

Reply:
Yeah, a bit confusing, but it holds.  See prev comment.